src/controller/python/test/test_scripts/base.py

```diff
@@ -53,6 +53,7 @@
 
 def TestFail(message, doCrash=False):
     logger.fatal("Testfail: {}".format(message))
+    doCrash = False
```

Comment:
Why is the input argument chanted? What is the purpose of such an argument if it is set to false anyway?

Reply:
Hang over from running some tests. Will revert.